test/services/ConfirmStakeIntent/update.test.ts

```diff
@@ -0,0 +1,59 @@
+import * as sinon from 'sinon';
+import ConfirmStakeIntentService from '../../../src/services/ConfirmStakeIntentService';
+import StubData from '../../test_utils/StubData';
+import SpyAssert from '../../test_utils/SpyAssert';
+import Repositories from '../../../src/repositories/Repositories';
+import Message from '../../../src/models/Message';
+
+interface TestConfigInterface {
+  repos: Repositories;
+}
+
+let config: TestConfigInterface;
+
+const Web3 = require('web3');
+
+describe('ConfirmStakeIntentService.update()', () => {
```

Comment:
Missing tests:  If stake request doesn't exist for a message hash it should fail?